hack/addons/update-nutanix-ccm.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
I thought about these again do we really need this file here if we're not going to support this with a CRS strategy?

Reply:
I added this originally thinking we can use this to capture the container images that will be needed, but we don't have that process yet, so removing it https://github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/pull/38/commits/abde55db57ac35e9465d2abbf37326ebd504289a

Thanks for the suggestion.